components/blocks/FilesToCards.vue

```diff
@@ -59,36 +62,213 @@
+              ><abbr :title="item.group | expandAcronym">
+                {{ item.group }}
+              </abbr></span
+            >
+            <h2 class="card-title">{{item.title}}</h2>
+            <div v-if="item.date">Updated: {{ item.date }}</div>
+            <span v-if="item.authors" class="small has-text-link"
+              >Authors: {{ item.authors.map((a) => a.name).join(', ') }}</span
+            >
+          </template>
+          <template #content>
+            {{ item.description }}
+          </template>
+          <template #footer>
+            <div v-if="item.links" class="link-group">
+              <a v-for="(link, type) in item.links" class="m-1 link-item d-button has-text-link" :href="link" :key="type">
+              <span>{{ type.toUpperCase() }}
+                <span class="icon ml-2">
+                  <i :class="['mdi', LINK_ICONS[type]]" />
+                  </span>
+                </span>
+              </a>
+            </div>
+          </template>
+        </DCard>
+      </div>
+    </div>
   </main>
 </template>
 
 <script>
 import DCard from '@/components/base/DCard.vue';
+import Multiselect from 'vue-multiselect';
 
 export default {
   components: {
     DCard,
+    Multiselect,
   },
   props: {
     data: {
       type: Array,
       required: true,
     },
   },
+  data: () => ({
+    ascending: true,
+    sortBy: [],
+    searchGroup: [],
+    sortOptions: [],
+    LINK_ICONS: {repository: "mdi-code-greater-than-or-equal", 
+              website: "mdi-link-variant", 
+              documentation: "mdi-book-open", 
+              publication: "mdi-newspaper",
+              doi: "mdi-book",
+              other: "mdi-dots-horizontal"}
+  }),
   computed: {
     filteredData() {
       return this.data.filter((d) => !d.hidden);
     },
+    sortedArray() {
+      let filtered = this.filteredData
+      if (this.searchGroup.length > 0) {
+        filtered = filtered.filter((card) => {
+          if (this.searchGroup) {
+            return this.searchGroup.includes(card.group);
+          } else {
+            return true;
+          }
+        });
+      }
+
+      // Sort by alphabetical order
+      if (this.sortBy.name === 'Title') {
+        filtered.sort((a, b) => {
+          const fa = a.title.toLowerCase();
+          const fb = b.title.toLowerCase();
+          if (fa < fb) {
+            return -1;
+          }
+          if (fa > fb) {
+            return 1;
+          }
+          return 0;
+        });
+      } else if (this.sortBy.name === 'Date') {
+        // Sort by date
+        filtered.sort((a, b) => {
+          return new Date(a.date) - new Date(b.date);
+        });
+      }
+
+      // Show sorted array in descending or ascending order
+      if (!this.ascending) {
+        filtered.reverse();
+      }
+
+      return filtered;
+    },
+  },
+  methods: {
+    cardTags() {
+      const tags = this.data.map((card) => card.group);
```

Comment:
I would make these (`cardTags` and `sortByOptions`) computed instead of methods

Reply:
if they are computed, how do i vbind them to multiselect? i tried this before and the internet told me to make them methods if i want to vbind them.